x/ccv/types/params.go

```diff
@@ -34,10 +34,10 @@ const (
 	// (and for consistency with other protobuf schemas defined for ccv).
 	DefaultHistoricalEntries = int64(stakingtypes.DefaultHistoricalEntries)
 
-	// In general, the default unbonding period on the consumer is one week less
+	// In general, the default unbonding period on the consumer is one day less
 	// than the default unbonding period on the provider, where the provider uses
 	// the staking module default.
-	DefaultConsumerUnbondingPeriod = stakingtypes.DefaultUnbondingTime - 7*24*time.Hour
+	DefaultConsumerUnbondingPeriod = stakingtypes.DefaultUnbondingTime - 24*time.Hour
```

Comment:
Why was this changed?

Reply:
No unbonding pausing anymore, so I guess no reason for it to be much smaller (?)

Reply:
Indeed